File: src/main/java/com/cyt/Hello.java
package com.cyt;

public class Hello {
    public static void main(String[] args) {
    //    System.out.println("Hello Wrold");
        String s = "abcde";
        System.out.println(s.charAt(3));
        System.out.println(s.length());
        System.out.println(s.substring(3));
        System.out.println(s.substring(1,4));
        Person p = new Person("Chris", 74.3f, 1.7f);
/*    p.weight = 74.3f;
    p.height = 1.7f;*/
        System.out.println(p.bmi());

/*    int age = 21;
    char c = '我';
    byte b = 120;
    float weight = 70.3f;
    float height = 1.7f;
    boolean adult = true;
    boolean enroll = false;
    String name = "Chris";*/

    }
}

File: src/main/kotlin/com/cyt/Hello.kt
package com.cyt

fun main() {
    //println("Hello KT")
    //Human().hello()
    val h = Human()
    h.hello()
    //var ＋ 變數 ＋ ：指定型態（一般不需指定，系統會自動選擇）
    //var age : Int = 21
    var age = 20
    age = 21
    var weight = 70.3
    val name : String
    name = "Chris"
}

class Human {
    fun hello () {
        println("Hello Kotlin")
    }
}